api/v1beta1/externaldns_types.go

```diff
@@ -304,7 +300,7 @@ type ExternalDNSAzureProviderOptions struct {
 	// }
 	//
 	// See
-	// https://github.com/kubernetes-sigs/external-dns/blob/master/docs/tutorials/azure.md
+	// https://github.com/kubernetes-sigs/external-dns/blob/226dbb931f7a2019810b3703aec096c4ea4f40ea/docs/tutorials/azure.md#configuration-file
```

Comment:
This is fine for now, but I hope in the future we will have some product documentation that we can reference instead.

Reply:
My goal was to show which fields can be put into the config. The previous link was a shot in the dark, this change makes the link at least useful. Btw, the config content is kinda opaque for the operator as we don't validate or enforce anything for the moment. But I agree the ultimate goal is to have this in the OCP docs.

Reply:
@jmanthei : Can you help us with adding of the Azure config file for EzternalDNS' Azure provider to the docs?

Reply:
@alebedev87 I'm happy to help- do you have links to existing config files for these that I can glean information from? How soon does this need to be done?

Reply:
We can take the upstream links as example. For instance [this one from the PR](https://github.com/kubernetes-sigs/external-dns/blob/226dbb931f7a2019810b3703aec096c4ea4f40ea/docs/tutorials/azure.md#configuration-file) for Azure.

Reply:
> How soon does this need to be done?

It's not urgent.